### ISSUE TITLE
Initiating prompt pay for RAMs with less complex payment process

### ISSUE THREAD

**Issue by azazime** [rchain/bounties#910]
### Benefit to RChain

From the general opinion of the last Colab  meeting which was held on the 16th August 2018 to which notification was earlier posted on the discord server encouraging RAMs to be part of the sociocratic process to determine the issue of prompt pay and possibly the elimination or total restructuring of the current invoicing procedure. Some of the major issues discussed during the meeting was the problem of **delay in payment,** the **complicated/outdated invoicing procedure** and the current **value of Rhocs/USD** as it relates to the payment of collaborators.
A consensus was reached and @azazime (Andrew) was chosen by sociocratic vote as delegate to represent RAMs in a quest to present such issues to the appropriate authorities and follow up to ensure the much needed changes are effected quickly enough. Please see [Log: noob rchain peer learning working group](https://docs.google.com/document/d/1HNIq_sBupkrtO3yuZT_5OlBWw8UEM3O9vNeQGtq9dpw/edit?usp=sharing)

A concerned member argued that if the blockchain technology favours the concept of economic freedom and fast processes, the Coop should also be able to abduct a better process to facilitate **prompt payment** and eventually a **choice of payment**. He continued by saying people work for different reasons for example; to pay taxes, to buy food or to pay their rent etc and they deserve prompt payment for job already done as delayed payments often times defeat the reason for their effort to work. He thought that its is not in the best interest of collaborators to be paid (for example) at the end of August or beginning of September for work done on July 1st, a thought which was an opinion shared by majority of members present at the meeting. Please see [Sociocratic Vote on current invoice system vs change in order to gain greater attendance to the matter](https://docs.google.com/document/d/1eREvzNJxrSCkJaZtFE4p28FPYJJyzfwqfhilX4o0Hfo/edit?usp=sharing)

**_PROPOSALS_**

* **Prompt Payment;
Immediately after the voting periods of every month which is on the 9th, then a one day period for cross checking for errors after which payments should be made on the 11th.

* RAMs should be able to conveniently decide how they should be paid.

* Scrapping the invoice procedures or modernization of its process**

More to follow as each proposals will be treated as issues on its own.

### Budget 

_Estimated Budget of Task:_ **$[????]**
_Estimated Timeline Required to Complete the Task:_ **[???]**

**Comment by tucsonblockchain** [comment]
I motion our delegate seek swift agreement from "the board" and a resolution is met - without delay in order to fast track this issue. If for some reason "the process" or discussion lingers beyond a reasonable amount of time, that Andrew, acting in full legal authority may offer a short term solution to initiate prompt pay, even if one payment at a time, manually to those that initiate a claim. I would say that a 5 day period is more than enough time to move towards this resolution and then if required the claims can begin and be paid one at a time to circumvent the slow process. At issue will be invoices submitted, acknowledged by receipt, and a budget to begin the payment process manually if necessary, via a claim process and by way of proper notice to RAMs. We must seek resolution immediately and our delegate should consider continually serving the motions to prove and flex the legal authority granted by common law of RAMs. This is not just a matter of prompt pay, or worker's rights, or blockchain technology, but it is a matter of equality and common law, and the exercise of equality before the law. If required our delegate may need to consider a declaration if there is no response or reasonable response that meets a specific deadline for resolution.

**Comment by dckc** [comment]
So let me get this straight... you're asking @lapin7 to turn payments around in 2 days rather than the usual 30 days for most businesses or the 15 days that the coop aspires to. And you offer no specifics as to how to achieve this. And you offer no support. And you claim that it is a right, rather than a privilege.

This is appalling, given his health situation. (#860)

Have you no shame?

**Comment by tucsonblockchain** [comment]
I think everybody wishes the best for @lapin7 but I also think that he would agree with both equality and equality before the law, and to insinuate "have you no shame", if that were directed at me, I have to say I am offering a solution, and in fact we granted a delegate full legal authority to seek resolution and a claim process, and perhaps if @lapin7 needs help, he too could seek or grant a delegate with the authority under the circumstances, but it is a bit confrontational to make such a comment when this amounts to business, nothing personal. While you bring up someone else up on a personal matter, do realize that others, maybe even RAMs may have personal issues and too require healthcare and need to pay it timely or maybe they might want to drink it all up at the bar on a timely basis before the bars close, all of which are personal choices, and this is about free choice and economic freedom, after all this is blockchain technology, hell maybe I might want to buy @lapin7 Mexican dinner on a timely visit if he were here, but to add insult to injury, come on man, let's get with the program. We are offering solutions and seeking resolution. 
I have zero shame in this process. I have zero shame in anything I have said or done, but I have meant what I have said and I'd say it for the next guy too. The issue title at the top of the page says it all, so I need not join in the personal commentary. 
Let us get back to the order of business so described.

**Comment by AyAyRon-P** [comment]
Full legal authority? A matter of equality? Common law? You're speaking about the bounty system that has worked for almost a year and positioning your comments like this has been a colossal issue for months which is simply not true. 

Who are the RAMs that are being represented? Jim, David, Brent, Aadi, Andrew, Jason, Valentine, Ibesoft, Owans, Ojimadu? I assure you the opinion of these ten members does not accurately reflect the opinion of the majority of RAMs or even most RAMs. Please refrain from speaking on behalf of the "majority", when again, this is not true. 

Up until 3 days ago there were no known concerns regarding the invoicing system. The original issue according to @philipandri002 was about timeliness, which is fair but complete restructuring of the rewards system is excessive and unrealistic.

**Comment by tucsonblockchain** [comment]
Yes, full legal authority granted. We served proper notice, we showed up, did you? We seek all RAMs support and full support of the delegate nominated. And we gave him a budget too to stand up for RAMs in this process. And yes, this is about equality before the law. This is blockchain technology.

**Comment by azazime** [comment]
@dckc you didn't understand the issue, plus we're ready to offer support as at where is needed so as to effect this change

**Comment by azazime** [comment]
I like that @jimscarver took out time to write in details, @dckc @AyAyRon-P you would agree with everyone else that the bounty system must improve (that is not to fault all that has been done prior to this time), we have to keep getting better collectively as a Coop. And like @tucsonblockchain said, This is blockchain technology.

**Comment by AyAyRon-P** [comment]
You gave 30 minutes of notice day of the meeting. That's hardly enough time to be convenient to anyone but the people who the meeting was clearly designed to serve and benefit. I invite you to lose the attitude as this is discussion not an argument. Stand up for RAMs? You say that like the members of the coop are oppressed, undervalued, underpaid, and need to fight for a revolution not a resolution. 

Yes, it is a blockchain technology. It's not a charity. It's an organization, and at the end of the day - it's a business. 

@azazime How can you offer support when you don't manage the payments and aren't privy to that information?

**Comment by allancto** [comment]
@azazime , all, thanks. We had a zoom discussion today [link?] in which there seemed to be consensus that there are 3 sub-issues here. 

1. Streamline invoice preparation to save time for Contributors and for administrative review.  (@allancto and @azazime will follow up)
2. Clarify practices in terms of usual and maximum delay for processing of invoices. (@allancto and @azazime will follow up)
3. Discussion of sociocratic voting, common law, legal authority seem to be not within the scope of this issue title and might be addressed in another issue or in other channels (@ICA3DaR5 and @tucsonblockchain will follow up) 

Thanks! -@allancto

**Comment by Keaycee** [comment]
The problem of late payment is  a collective issue. We must understand that it does affect every one member of the community. But the system has improved since we employed the new webApp, although not fully automated.  Like @jimscarver said,

>  A simple webapp replacing the declaration form can make it as simple as hitting the PAY ME button if there is nothing you want to change.

I believe soon, the payment  process will be much more simpler and fast, but we all have to be patient.
@allancto I'm would like to assist on 1&2.

**Comment by ICA3DaR5** [comment]
Discussion on issues #904 and #910: https://youtu.be/tHzuVqEyMZ4

**Comment by allancto** [comment]
@Keaycee, all, please see #912, Improved Invoice Submission Process and Protocol. Please consider contributing! Particularly if you are able to help with development. Timeline is for prototype which will be sufficient to use as an (optional) alternative for August voting (Sept 9). Developers wanted!

**Comment by azazime** [comment]
Thanks @allancto for a prompt response please @David405 @Valentine-Mario @dckc @ibesoft11 @ICA3DaR5 @jimscarver see #912 (help needed)

**Comment by dckc** [comment]
I looked at #912.

If there is something in particular you want me to do, please be more clear.

**Comment by azazime** [comment]
@dckc nice you looked at #912 because we would need your assistance as well as your support, thanks.

**Comment by ICA3DaR5** [comment]
Some more discussion on issues #904, #910 and #912

**Comment by dckc** [comment]
I propose that this is subsumed by #912.

Anybody who disagrees, feel free to explain why and re-open the issue.

**Comment by tucsonblockchain** [comment]
open

**Comment by dckc** [comment]
@philipandri002 writes:
> Immediately after the voting periods of every month which is on the 9th, then a one day period for cross checking for errors after which payments should be made on the 11th.

Yes, that's been said before. It's not actionable. Nobody here has been asked to make that happen. I can't imagine anyone who can make it happen. There's a coop staff person who does their level best to get payments out in a timely fashion. It's not reasonable to say "do it in one day instead of two to four weeks". So unless you have new information, this is not reason to re-open the issue.

> RAMs should be able to conveniently decide how they should be paid.

As @jimscarver pointed out, payment in bitcoin or ether has been tried; it makes the process neither more prompt nor simple. So it is not consistent with the title of this issue. If you have a proposal for a next step on making it happen, I suggest opening a new issue.

> Scrapping the invoice procedures

not a legal possibility, as @ICA3DaR5 points out

> or modernization of its process**

That is exactly the scope of #912.

> @azazime ... I'm sure the RAMs are solidly behind you to finish this daring task.

For one thing, speak for yourself. For another, the only thing I have seen @azazime offer to do is to follow up on #912.

I see no new information that merits re-opening this issue.

**Comment by tucsonblockchain** [comment]
https://docs.google.com/document/d/1Is9waQPF9xRCLVWJbkA4csR4KEdjWOxrfcTBFDUOYhw/edit?usp=sharing

**Comment by azazime** [comment]
There was alot of brainstorming for hours on various issues ranging from prompt payment, rewards for work, Invoicing processes and Rhocs. We made suggestions which prompted issue #912  by @Allanc  and alot of follow ups was made including getting @margosha(project manager) to give an update on progress made during one of the colab's meeting on zoom. Also we created the delegates channel on the Colab server which will serve as an update link. This issue is a fuse to #912
